archivist/cmds/usage/main.py

```diff
@@ -0,0 +1,27 @@
+# pylint:  disable=missing-docstring
+
+from logging import getLogger
+from sys import exit as sys_exit
+from sys import stdout as sys_stdout
+
+from archivist.archivist import Archivist
+from archivist.cmds.usage.run import run
+from archivist.parser import common_parser
+
+LOGGER = getLogger(__name__)
+
+
+def main():
+    parser = common_parser(description="RKVST Diagnostic Tool")
+
+    args = parser.parse_args()
+
+    arch = Archivist(
```

Comment:
Ah - I see the endpoint  function is not being used.
Is there a reason? 
it sets up logging etc etc and allows both types of authorisation - token or app registration...

Reply:
My mistake, the entrypoint was added to setup.py but I wasn't sure where to implement the endpoint function, I somehow missed it in the cmds/runner example. Change made!